### DISCUSSION TITLE
Airspy HF+ discovery not detected (SOLVED)

### DISCUSSION THREAD

**Discussion by f4bpp** [AlexandreRouma/SDRPlusPlus#1127] [General]
Hi Alexandre,
I've just received my Airspy HF+ discovery SDR receiver. I've installed all the drivers (included soapy sdr module airspy). The device is well recognized by GQRX but not by SR++. 
I select Airspy on the top dropdown menu but in the dropdown menus just below, the lists are empty.
Do I miss something ?
I'm using the last night build for Ubuntu Jamy.

Thanks for help.

David.

**Comment by starvald** [comment]
it should be Airspy HF+ in sources rather than just Airspy, do you have have airspyhf and libairspyhf1 installed?

also its worth rebooting after installing, there should be udev rules to allow local user access.

**Reply by f4bpp** [reply]
GQRX detect the SDR as AirspyHF device and the device string is "aispyhf=0". The input rate is set to 768000 and the sample rate to 768.000 ksps.

**Reply by f4bpp** [reply]
![QGRX](https://github.com/AlexandreRouma/SDRPlusPlus/assets/16517945/64cb9404-9240-475b-b130-1dde9611a1fc)

**Comment by f4bpp** [comment]
Both airspyhf and libairspyhf1 are installed and there is no Airspy HF+ in the menu. About the udev rules, I know how it works but I don't know the rules to add to make it work.

**Reply by starvald** [reply]
does running airspyhf_info return anything?

if it picks it up without needing sudo then that side of it is ok.

**Comment by f4bpp** [comment]
The rules 52-airspyhf.rules is ok :
ATTR{idVendor}=="03eb", ATTR{idProduct}=="800c", SYMLINK+="airspyhf-%k", MODE="660", GROUP="plugdev"

and airspyhf_info without sudo return that :
S/N: 0x3652975D4ADE398D
Part ID: 0x00000002
Firmware Version: R3.0.6-CD
Available sample rates: 768 kS/s 384 kS/s 256 kS/s 192 kS/s

**Comment by starvald** [comment]
not sure whats up then, i set up 22.04 in a virtual machine only last week without issue.
i'm sure Alexandre will know when he's back online.

also there is a newer firmware for it to unlock 912 kS/s bandwidth.

from mine
```
user@xfce:~$ airspyhf_info 
AirSpy HF library version: 1.6.8

S/N: 0x3652A85DE9C746F7
Part ID: 0x00000002
Firmware Version: R3.0.7-CD
Available sample rates: 912 kS/s 768 kS/s 456 kS/s 384 kS/s 256 kS/s 192 kS/s
```

**Reply by f4bpp** [reply]
Thanks starvald. I've updated mine in case of...

AirSpy HF library version: 1.7.1

S/N: 0x3652975D4ADE398D
Part ID: 0x00000002
Firmware Version: R3.0.7-CD
Available sample rates: 912 kS/s 768 kS/s 456 kS/s 384 kS/s 256 kS/s 192 kS/s

But SDR++ still don't detect anything. 

What is the version of SDR++ you are using ? Mine is nightly build for Ubuntu Jamy.

**Comment by starvald** [comment]
i installed sdrpp from the ubuntu repos v1.1.0

**Comment by starvald** [comment]
it should be enabled by default but check the module is added in the sdrpp module manager.
![Screenshot_20230708_140438](https://github.com/AlexandreRouma/SDRPlusPlus/assets/20885830/893be16f-3f91-4e1f-96d1-e2f7ae3205ff)

**Reply by f4bpp** [reply]
Thank's Alexandre. Problem solved. I have to remove Airspy Source then add AispyHF Source. I'm happy !
